packages/reports/src/Constants/data-table-constants.tsx

```diff
@@ -279,12 +286,7 @@ export const getMultiplierOpenPositionsColumnsTemplate = ({
             if (is_footer) return localize('Total');
 
             return (
-                <MarketSymbolIconRow
-                    action={cell_value}
-                    key={row_obj.id}
-                    payload={row_obj.contract_info}
-                    should_show_multiplier={false}
-                />
+                <MarketSymbolIconRow key={row_obj.id} payload={row_obj.contract_info} should_show_multiplier={false} />
```

Comment:
in the `packages/reports/src/Components/market-symbol-icon-row.tsx` path the `show_description` should be optional in the `TMarketSymbolIconRow` type

Reply:
the  `show_description?: boolean;` in `TMarketSymbolIconRow` is already optional.